src/widgets/header/ui/burger-menu.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import { AlignJustify, X } from 'lucide-react';
+import Link from 'next/link';
+import { Socials } from '~/features/contacts';
+import { LogoBig } from '~/shared/ui/assets';
+import {
+  Sheet,
+  SheetClose,
+  SheetContent,
+  SheetTrigger,
+} from '~/shared/ui/sheet';
+
+const links = [
+  { label: 'Store', path: '/store' },
+  { label: 'News', path: '/news' },
+  { label: 'Contact', path: '/contact' },
+] as const;
+
+export const BurgerMenu = () => {
+  return (
+    <Sheet>
+      <SheetTrigger className='xl:hidden'>
+        <AlignJustify className='stroke-1 transition-colors hover:text-black' />
+      </SheetTrigger>
+
+      <SheetContent
+        side='right'
+        className='w-full border-0 bg-card p-0 md:w-[23.5rem]'
+      >
+        {/* Header */}
```

Comment:
Instead of comments you can separate this into components

Reply:
remove comments